libretroshare/src/gxs/rsgxsdataaccess.cc

```diff
@@ -833,7 +833,7 @@ void RsGxsDataAccess::processRequests()
 			}
 			else
 			{
-				req->status = FAILED;
+				delete req;//req belongs to no one now
 				mPublicToken[req->token] = FAILED;
```

Comment:
do it the other way, otherwise req->token will possibly cause an access to freed memory,

Reply:
Oups.
Done